src/elements/content-sidebar/stories/MetadataSidebarRedesign.stories.tsx

```diff
@@ -1,8 +1,14 @@
 import { type StoryObj } from '@storybook/react';
 import { fn, userEvent, within } from '@storybook/test';
 import React, { type ComponentProps } from 'react';
+import { http, HttpResponse } from 'msw';
 import MetadataSidebarRedesign from '../MetadataSidebarRedesign';
 import ContentSidebar from '../ContentSidebar';
+import {
+    mockEnterpriseMetadataTemplates,
+    mockFileRequest,
+    mockMetadataInstances,
+} from './__mocks__/MetadataSidebarRedesignedMocks';
 
 const fileIdWithMetadata = global.FILE_ID;
```

Comment:
shouldn't we import the variable from mock data now or it doesn't matter here?

Reply:
I think it doesn't matter much. And if we start using this mocks for all stories, we can stop relying data coming from test account all together.